app/package.json

```diff
@@ -13,42 +13,45 @@
     "start": "NODE_ENV=development expo start",
     "android": "expo run:android",
     "ios": "expo run:ios",
+    "postinstall": "node postinstall.js",
```

Comment:
can you quickly explain if there is a reason for this script? Any dependency that requires it now? If so, please add it to the README → installation part.

Reply:
It appears this was one of the bumps in the road that is no longer necessary. I removed it